fix(git): allow the command to pass on windows

yoannmoinet opened the pull request:
When a command has spaces in windows, it seems that it doesn't consider spaces as part of the string unless it's wrapped in double-quote `"`.

Single-quotes `'` are not considered a string.

stevemao commented:
Thanks @yoannmoinet , really appreciate your time on this. We need tests on this. If you have time can you add some to prove `format` works? Sorry I think this is probably the only test case I forgot.

yoannmoinet commented:
Sure thing, I'll see what I can do for these tests.

yoannmoinet commented:
Hey, so I've fixed another problem I had on windows with `from` and `to` not being escaped correctly on windows when adding `^` to the selector (probably with others too).
The solution was to also wrap those in double-quotes `"`.

I've added a test, just to be sure that a simple command passes without errors.
It should tackle this kind of errors first.

For the format validation, I think it's a little beyond the scope of this PR.
So I'll see what I can do in another PR if I have the time to tackle this.
I don't even know how to correctly validate this. There are way too many placeholders to check.
Do you have a plan for this?

As a side note, `jscs *.js` in your npm script  ` npm test` doesn't run on Windows. 
`jscs` doesn't recognize wildcards  ` *` and only takes filenames in flat list.

stevemao commented:
Thank you very much for this. Will have a look soon.

For format validation we can add some dummy commits to the git in the tmp folder and see if we can get what we want. (don't validate the template, as it may not get what we want under different system) we could put author name in the format and see if the returned log contains author name, etc.

> `jscs *.js` in your npm script  npm test doesn't run on Windows. 

So does it mean windows does not support glob? or package.json does not support glob on windows? If so we shouldn't contain glob in package.json?

stevemao commented:
Merged, really appreciate it. Thanks @yoannmoinet

yoannmoinet commented:
I have no idea for glob on windows.

In the merge, you missed the two [latest](https://github.com/yoannmoinet/git-raw-commits/commit/c63ca529e0e8550dd2abd26d7088f49f77cd6395) [commits](https://github.com/yoannmoinet/git-raw-commits/commit/dcbf345c7714aaa644935a0f0695f596b574d7d1), was that intentional?

stevemao commented:
All your three commits are in the master branch. I rebased them just like how angular manages their repos.

yoannmoinet commented:
Indeed... my bad, github was giving me a weird message about the closed pull-request.
